fog/report/validation/src/lib.rs

```diff
@@ -10,131 +10,103 @@
 //! mobilecoin transactions with fog recipients on an embedded device like a
 //! tiny hardware wallet, that doesn't have threads and won't have rust std.
 
-#![deny(missing_docs)]
```

Comment:
Not sure if there's a specific reason this is needed. It was causing trouble when I was building and just removed it for the time being. Happy to re-add it if it's important.

Reply:
Its important to force developers to add doc strings, can we add doc strings wherever they are missing so it will build with this?

Reply:
figured that's what it was, but it didn't seem consistent with what was already there. I'll look into it and add doc updates where necessary :)

Reply:
aaah okay, it applies this rule to the entire file, duh